plugin/route53/setup.go

```diff
@@ -35,7 +35,9 @@ func init() {
 
 func setup(c *caddy.Controller, f func(*credentials.Credentials) route53iface.Route53API) error {
 	keys := map[string]string{}
-	credential := credentials.NewEnvCredentials()
+	sharedProvider := &credentials.SharedCredentialsProvider{}
```

Comment:
Why do you need this here? Don't you want to append shared provider only when `credentials` stanza is present? This comment also applies to line 98 below.

Reply:
I see. Could you add comment about this so that somebody doesn't remove it later by mistake.